etc/picongpu/taurus-tud/knl_picongpu.profile.example

```diff
@@ -0,0 +1,42 @@
+# Text Editor for Tools #######################################################
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load cmake/3.9.0
+module load git
+module load intelmpi/2017.2.174
+
+# Compilers ###################################################################
+### ICC
+module load intel/2017.2.174
+# Boost needs to be build yourself for now!
+# These paths need to be adjusted.
+export LD_LIBRARY_PATH=<BOOST_INSTALLATION_DIR>:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=<BOOST_INSTALLATION_DIR>/lib:$LD_LIBRARY_PATH
+export BOOST_ROOT=<BOOST_INSTALLATION_DIR>
```

Comment:
Could you move this line 3 lines up and than use `BOOST_ROOT` for the `LD_LIBRARY_PATH`

Reply:
Hm. Makes sense.